docs/networking.md

```diff
@@ -0,0 +1,160 @@
+    - Emphasize stateful rules, focusing on allow-listing required ingress traffic.
+
+  - **NACLs:**
+    - Used sparingly as a defense-in-depth measure, particularly in Data subnets, to enforce segmentation.
+    - ASEA encourages review and tailoring based on specific security requirements. -->
+
+- **Best Practices for Configuring Security Groups in Workload VPCs:**
+  - Security Groups are recommended as the primary data-plane isolation mechanism.
+  - Sample security groups provided as a balance between security, ease of operations, and frictionless development, with the expectation that customers will refine them based on their specific security needs.
+  - Only open the necessary ports, IP ranges/addresses and grant the minimum required permissions to each security group to minimize exposure to vulnerabilities.
+
+### Exposing services to the internet
+
+Generally, in the ASEA we recommend one of two methods of exposing services to the internet:
+- API Gateway
+- Application Load Balancer (ALB)
+
+Making strategic choices between AWS API Gateway and ALBs is essential for optimizing cloud architecture. API Gateway is the preferred option for internet exposure, catering to modern applications using RESTful APIs and serverless computing. ALBs are should only be used for supporting legacy applications, and require integration support from the Public Cloud team.
+
+**Create an API in API Gateway:**
+For general instructions on how to cerate an API gateway and safely expose it to the internet please see this [AWS documentation on HTTP APIs](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api.html). For ASEA specific examples deployed using Terraform via GitHub Actions please see our [serverless, or container based sample applications](./deploy-an-app-to-the-aws-landing-zone.md#sample-applications). Creating resources via Terraform/ automation is always preferred.
+
+**Benefits of Using API Gateway and VPC Link:**
+- **Security and Isolation:** API Gateway and VPC Link provide a secure and isolated connection between your internet-facing API and backend services within a VPC. AWS WAF can also be used alongside your API gateway. 
+    
+- **Scalability:** API Gateway scales automatically to handle varying levels of traffic, ensuring the availability of your internet-facing services.
+    
+- **Managed Service:** API Gateway is a fully managed service, reducing operational overhead and allowing you to focus on building and improving your APIs.
+
+**When to use ALBs:**
+While API Gateway is recommended for modern applications, there is still a use case for ALBs with legacy applications where it may be difficult or impossible to use an API Gateway. If your application is unable to accommodate an API Gateway, then please reach out to the [Cloud Pathfinder Team](https://digital.gov.bc.ca/cloud/services/public/get-support/#contact) for integration support.
```

Comment:
```suggestion
While API Gateway is recommended for modern applications, there is still a use case for ALBs with legacy applications where it may be difficult or impossible to use an API Gateway. If your application is unable to accommodate an API Gateway, then please reach out to Cloud.Pathfinder@gov.bc.ca for integration support.
```